<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Microbiome resources</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>



<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Microbiome resources</h1>

</div>


<p>A collection of good journal articles related to methods used to study the microbiome. Contributions, in the form of pull requests on the github page are welcome. Please include a description of the paper indicating its importance to the field.</p>
<div id="must-reads" class="section level2">
<h2>Must Reads</h2>
<p><em>Foundational methods paper describing how to do amplicon sequencing data using paired-end sequencing on the MiSeq.</em></p>
<ul>
<li>Kozich JJ, Westcott SL, Baxter NT, Highlander SK, Schloss PD. Development of a Dual-Index Sequencing Strategy and Curation Pipeline for Analyzing Amplicon Sequence Data on the MiSeq Illumina Sequencing Platform. Appl Environ Microbiol. American Society for Microbiology (ASM); 2013;79: 5112–5120. <a href="http://aem.asm.org/content/early/2013/06/17/AEM.01043-13">link</a></li>
</ul>
<p><em>The ‘kitome’ paper. Why blanks are absolutely essential</em></p>
<ul>
<li>Salter SJ, Cox MJ, Turek EM, Calus ST, Cookson WO, Moffatt MF, et al. Reagent and laboratory contamination can critically impact sequence-based microbiome analyses. BMC Biology 2015 13:1. BioMed Central; 2014;12: 1. <a href="http://bmcbiol.biomedcentral.com/articles/10.1186/s12915-014-0087-z">link</a></li>
</ul>
<p><em>Why it’s really not a good idea to ‘rarefy’. Also good evidence for using DESeq2 for identification of differentially abundant OTUs</em></p>
<ul>
<li>McMurdie PJ, Holmes S. Waste Not, Want Not: Why Rarefying Microbiome Data Is Inadmissible. McHardy AC, editor. PLoS Comp Biol. Public Library of Science (PLoS); 2014;10: e1003531. <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003531">link</a></li>
</ul>
</div>
<div id="good-blog-posts" class="section level2">
<h2>Good blog posts</h2>
<p><em>Note that this is not a list of good blog sites - just single blog posts that are informative and useful</em></p>
<p>Why do I have such a large distance matrix? - The mothur blog by Pat Schloss <a href="http://blog.mothur.org/2014/09/11/Why-such-a-large-distance-matrix/">link</a></p>
<p><em>A rather humorous blog post by Pat Scloss on why you should sequence way more than you think you need, and throw out anything without good quality and overlapping pairs</em></p>
</div>
<div id="other-collections" class="section level2">
<h2>Other Collections</h2>
<p>All of <a href="https://microbiomedigest.com/">Microbiome Digest</a> is particularly good but here are a couple of useful collections from that website.</p>
<ul>
<li><a href="https://microbiomedigest.com/microbiome-papers-collection/microbiome-techniques/extraction-and-amplification/">Extraction and Amplification</a></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
